config/zones/TH.yaml

```diff
@@ -19,18 +19,108 @@ capacity:
 contributors:
   - PPsyrius
 emissionFactors:
-  direct:
-    unknown:
-      _url: https://docs.google.com/spreadsheets/d/16j5BVc4L34BDqVE0GeDBv0P3zQupT6zU77Kj5UK7GjE/edit?usp=sharing
-      source: IEA 2020; assumes 9.1% biomass, 19.3% coal, 64.7% gas, 2.5% hydro, 2.6%
-        solar, 1.7% wind, 0.1% other
-      value: 386.645
```

Comment:
I don't think these should necessarily be removed. Is there any  particular reason for it?

Reply:
I assume it should be removed as it was from a different calculation, but yeah - will put them back in.